src/read.c

```diff
@@ -2886,11 +2838,6 @@ static Int InitKernel (
 static Int InitModuleState(void)
 {
     STATE(ErrorLVars) = (UInt **)0;
-    ReaderState()->StackNams = NEW_PLIST(T_PLIST, 16);
-    ReaderState()->ReadTop = 0;
-    ReaderState()->ReadTilde = 0;
-    ReaderState()->CurrLHSGVar = 0;
-    ReaderState()->CurrentGlobalForLoopDepth = 0;
```

Comment:
There is no per-thread `ReaderState` as part of the `GAPState` anymore, so the above can go.